From 411c15a88b4f552ed6d83e20796e8a28817d5935 Mon Sep 17 00:00:00 2001
From: Maurizio Lombardi <mlombard@redhat.com>
Date: Mon, 8 Sep 2014 11:07:38 -0400
Subject: [scsi] bnx2fc: remove unused variable hash_table_size

Message-id: <1410174472-14981-4-git-send-email-mlombard@redhat.com>
Patchwork-id: 91633
O-Subject: [RHEL7.1 e-stor PATCH 03/17] bnx2fc: remove unused variable hash_table_size
Bugzilla: 1089399
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

hash_table_size is not used by the bnx2fc_free_hash_table() function.

Signed-off-by: Maurizio Lombardi <mlombard@redhat.com>
Acked-by: Eddie Wai <eddie.wai@broadcom.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit b044d1498cbfef5e11c9c390288f4b841c9b328c)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/bnx2fc/bnx2fc_hwi.c b/drivers/scsi/bnx2fc/bnx2fc_hwi.c
index 46a3765..261af2a 100644
--- a/drivers/scsi/bnx2fc/bnx2fc_hwi.c
+++ b/drivers/scsi/bnx2fc/bnx2fc_hwi.c
@@ -1966,12 +1966,9 @@ static void bnx2fc_free_hash_table(struct bnx2fc_hba *hba)
 {
  int i;
  int segment_count;
- int hash_table_size;
  u32 *pbl;
 
  segment_count = hba->hash_tbl_segment_count;
- hash_table_size = BNX2FC_NUM_MAX_SESS * BNX2FC_MAX_ROWS_IN_HASH_TBL *
-  sizeof(struct fcoe_hash_table_entry);
 
  pbl = hba->hash_tbl_pbl;
  for (i = 0; i < segment_count; ++i) {
-- 
1.7.1